gatsby-config.js

```diff
@@ -74,12 +74,6 @@ module.exports = {
         ],
       },
     },
-    {
-      resolve: 'gatsby-plugin-netlify-cms',
```

Comment:
What did this plugin do before removal, anything?

Reply:
This plugin is what made things happen when you went to /admin